src/backend/cuda/utility.cpp

```diff
@@ -0,0 +1,35 @@
+/*******************************************************
+ * Copyright (c) 2019, ArrayFire
+ * All rights reserved.
+ *
+ * This file is distributed under 3-clause BSD license.
+ * The complete license agreement can be obtained at:
+ * http://arrayfire.com/licenses/BSD-3-Clause
+ ********************************************************/
+
+#include <err_cuda.hpp>
+#include <utility.hpp>
+
+using std::make_pair;
+using std::pair;
+
+namespace cuda {
+
+int interpOrder(const af_interp_type p) noexcept {
```

Comment:
should probably be included in a separate header.

Reply:
It is a helper used in three different locations, as of now this is a common location we have. we can refactor the header name later if that is something we want to do.